dask_jobqueue/core.py

```diff
@@ -144,8 +205,8 @@ def __init__(self,
 
     def job_script(self):
         """ Construct a job submission script """
-        self.n += 1
-        template = self._command_template % {'n': self.n}
+        self._n += 1
+        template = self._command_template % {'n': self._n}
```

Comment:
"-%(n)d" has been removed from self._command_template (l.137), so we don't need this line if I'm not mistaken.